common/db.go

```diff
@@ -394,10 +395,29 @@ func (db *DB) ChainID() (*big.Int, error) {
 }
 
 func (db *DB) SetChainID(id *big.Int) error {
-	if err := db.updateKVStore("chainID", id.String()); err != nil {
-		return err
+	return db.updateKVStore("chainID", id.String())
+}
+
+func (db *DB) CurrentRound() (*big.Int, error) {
```

Comment:
`LastInitializedRound()` (with the key name as `lastInitializedRound`) so that all types that currently depend on an interface that defines `LastInitializedRound()` can just plug in `DB` instead?